src/fastcs/backends/epics/ioc.py

```diff
@@ -27,6 +27,25 @@ class EpicsIOCOptions:
     terminal: bool = True
 
 
+DATATYPE_NAME_TO_RECORD_FIELD = {
+    "prec": "PREC",
+    "units": "EGU",
+    "min": "DRVL",
+    "max": "DRVH",
+    "min_alarm": "LOPR",
+    "max_alarm": "HOPR",
+    "znam": "ZNAM",
+    "onam": "ONAM",
+}
+
+
+def datatype_to_epics_fields(datatype: DataType) -> dict[str, Any]:
+    return {
+        DATATYPE_NAME_TO_RECORD_FIELD[field]: value
+        for field, value in asdict(datatype).items()
+    }
```

Comment:
This will raise a confusing error if a field is missing. Could we catch the exception and provide a useful message?

Reply:
I'll make it so that it doesn't add anything if the `field` isn't defined with an epics name in the dictionary. It may only be possible to implement some fields in some backends.